src/components/routineSingle.js

```diff
@@ -0,0 +1,27 @@
+const RoutineSingle = () => {
+
+
+
+
+    return (
+        <>
+            <h3>Creator:</h3>
+            <div>
+                {routine.creatorName}
+            </div>
+            <h3>Name:</h3>
+            <div>
+                {routine.name}
+            </div>
+            <h3>Goal:</h3>
+            <div>
+                {routine.goal}
+            </div>
+        </>
```

Comment:
It would be better to change the react fragment wrapping all the information into a div with a className='routine-single'. This will help you with styling the page later down the line.